front_end/entrypoints/inspector_main/CohtmlPanel.ts

```diff
@@ -0,0 +1,226 @@
+    if (mainTarget)
+    {
+      this.cohtmlDebugModel = mainTarget.model(SDK.CohtmlDebugModel.CohtmlDebugModel);
+    }
+
+    this.drawMetaDataSetting = Common.Settings.Settings.instance().moduleSetting('drawMetaData');
+    this.continuousRepaintSetting = Common.Settings.Settings.instance().moduleSetting('continuousRepaint');
+
+    this.appendCheckbox(
+        i18nString(UIStrings.paintFlashing), i18nString(UIStrings.highlightsAreasOfThePageGreen),
+        Common.Settings.Settings.instance().moduleSetting('showPaintRects'));
+    this.appendCheckbox(
+        i18nString(UIStrings.redrawFlashing), i18nString(UIStrings.highlightsAreasOfThePageRed),
+        Common.Settings.Settings.instance().moduleSetting('showRedrawRects'));
+
+    this.contentElement.createChild('div').classList.add('panel-section-separator');
+
+    this.appendCheckbox(UIStrings.toggleMetaDataTitle, UIStrings.toggleMetaDataDesc,
+                        Common.Settings.Settings.instance().moduleSetting('drawMetaData'));
+
+    this.appendCheckbox(UIStrings.toggleContinuousRepaintTitle, UIStrings.toggleContinuousRepaintDesc,
+                        Common.Settings.Settings.instance().moduleSetting('continuousRepaint'));
+
+    this.createSimpleButton(UIStrings.dumpDomTitle, UIStrings.dumpDomDesc, 'dumpDOM');
+    this.createSimpleButton(UIStrings.dumpStackingContextTitle, UIStrings.dumpStackingContextDesc, 'dumpStackingContext');
+    this.createSimpleButton(UIStrings.dumpUsedImagesTitle, UIStrings.dumpUsedImagesDesc, 'dumpUsedImages');
+    this.createSimpleButton(UIStrings.captureBackendTitle, UIStrings.captureBackendDesc, 'captureBackend');
+    this.createSimpleButton(UIStrings.captureRendTitle, UIStrings.captureRendDesc, 'captureRend');
+    this.createSimpleButton(UIStrings.capturePageTitle, UIStrings.capturePageDesc, 'capturePage');
+
+    this.contentElement.createChild('div').classList.add('panel-section-separator');
+
+    this.createSimpleButton(UIStrings.clearCachedUnusedImagesTitle, UIStrings.clearCachedUnusedImagesDesc, 'clearCachedUnusedImages');
+
+    this.createButton(UIStrings.getSystemCacheTitle, UIStrings.getSystemCacheDesc, () => {
+      if (this.cohtmlDebugModel)
+      {
+        const cacheStats = this.cohtmlDebugModel.getSystemCacheStats();
+        cacheStats?.then((obj) => {
+          console.log('Alive Images:', obj?.stats);
+        });
+      }
+    });
+  }
+
+  static instance(opts: {
+    forceNew: boolean|null,
+  } = {forceNew: null}): CohtmlPanelView {
+    const {forceNew} = opts;
+    if (!cohtmlPanelViewInstance || forceNew) {
+      cohtmlPanelViewInstance = new CohtmlPanelView();
+    }
+
+    return cohtmlPanelViewInstance;
+  }
+
+  private createSimpleButton(title: string, description: string, method: any) {
+    this.createButton(title, description, () => {
+      if (this.cohtmlDebugModel && (this.cohtmlDebugModel as any)[method]) {
+        (this.cohtmlDebugModel as any)[method]();
+      }
```

Comment:
```suggestion
      const cohtmlDebugMode = this.cohtmlDebugModel as any;
      if (this.cohtmlDebugModel && this.cohtmlDebugModel[method]) {
        this.cohtmlDebugModel[method]();
      }
```

Reply:
I am not sure if it will work but you can try to add an index signature to the CohtmlDebugModel class like this
```
class Foo implements IRawParams {
    [k: string]: any;
}
```
and test if you need to specify the `any` type here.

Reply:
yep, it worked, I've gotten rid of the one `any` usage